doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,106 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
+
+1. Using the `kexec` tool as described in link:../../usr/share/rear/conf/templates/RESULT_usage_RAMDISK.txt[RESULT_usage_RAMDISK.txt],
+   if you have access to a running Linux instance.
+
+2. Using the `zipl` tool for management of boot devices on s390/s390x.
+
+=== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM with `zipl`
+
+After creating the rescue image with `rear mkrescue`, move the kernel and
+initramfs to a directory (denoted by `REAR_DIRECTORY` in this example) on the
+target device to boot from.
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option.
+      If you want to use a tape device, see `zipl(8)` to use the `--tape`
+      option instead. Note that the `--tape` option with ReaR is untested so
+      contributions to this guide are welcome! Also, it is not possible to use
+      `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory!
+
+Use the following command to make the device with `REAR_DIRECTORY` bootable
+with `zipl`:
+```
+# zipl --target=BOOTMAP_INSTALL_DIRECTORY \
+       --parameters='root=/dev/ram0 ro' \
+       --image=REAR_DIRECTORY/kernel-HOSTNAME \
+       --ramdisk=REAR_DIRECTORY/initramfs-HOSTNAME.img
+Run /lib/s390-tools//zipl_helper.device-mapper BOOTMAP_INSTALL_DIRECTORY
+Building bootmap in 'BOOTMAP_INSTALL_DIRECTORY'
+Adding IPL section
+  initial ramdisk...: REAR_DIRECTORY/initramfs-HOSTNAME.img
+  kernel image......: REAR_DIRECTORY/kernel-HOSTNAME
+  kernel parmline...: 'root=/dev/ram0'
+  component address:
+    internal loader.: 0x0000a000-0x0000dfff
+    parameters......: 0x00009000-0x00009fff
+    kernel image....: 0x00010000-0x00cdbfff
+    parmline........: 0x00cdc000-0x00cdcfff
+    initial ramdisk.: 0x00cf0000-0x10980fff
+    environment blk.: 0x00cec000-0x00cecfff
+Preparing boot device for CCW- and LD-IPL: dasda.
+Done.
+```
+You must set `BOOTMAP_INSTALL_DIRECTORY` to any directory on the same device
```

Comment:
You don't set `BOOTMAP_INSTALL_DIRECTORY` to anything as it is not a variable. You could write "replace `BOOTMAP_INSTALL_DIRECTORY` by (...)", or keep it and replace `BOOTMAP_INSTALL_DIRECTORY` in your examples by `$BOOTMAP_INSTALL_DIRECTORY`, but I believe it is better to use a real directory here. See above.

Reply:
The guide now uses explicit paths as suggested in https://github.com/rear/rear/pull/3158#discussion_r1509456636.